flink-runtime-web/pom.xml

```diff
@@ -162,11 +162,6 @@ under the License.
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-jar-plugin</artifactId>
 				<executions>
-					<execution>
-						<goals>
-							<goal>test-jar</goal>
-						</goals>
-					</execution>
```

Comment:
That is in general obsolete because it doesn't provide additional configuration, isn't it?

Reply:
It's obsolete because no other module requires the test-jar.

Reply:
the test-jar goal is not obsolete in general, because we only create a test jar if this goal is configured.